### DISCUSSION TITLE
Does neorg table supports multi-row/multi-column cells？

### DISCUSSION THREAD

**Discussion by Ninlives** [nvim-neorg/neorg#404] [General]
The first example in the spec seems to produce a multi-column cell but I don't understand bhow the syntax works. Also, it seems not possible to declare more complex table structures. Is this feature currently supported? If not, is this feature on the roadmap? Thanks!

**Comment by mrossinek** [comment]
Hi there! Sorry about the long delay here...

We have not yet designed a proper table syntax. There are some preliminary ideas implemented in https://github.com/nvim-neorg/tree-sitter-norg-table but these are by no means what the actual syntax might become. This is planned for 0.2 so stay tuned! :tada:

**Reply by Ninlives** [reply]
Really looking forward to this! Thanks for your effort!

**Comment by pysan3** [comment]
Seems to be supported with
```txt
: A1-A2 : multirow cell
: > : b1
: > : c1
: _> : b2
: > : c2
```

Tho no parser or visualizer implemented yet.
